src/dmap_import/pipeline.py

```diff
@@ -65,6 +67,38 @@ def validate_environment(
     process_logger.log_complete()
 
 
+def check_for_parallel_tasks() -> None:
+    """
+    Check that that this task is not already running on ECS
+    """
+    process_logger = ProcessLogger("check_for_tasks")
+    process_logger.log_start()
+
+    client = boto3.client("ecs")
+    dmap_ecs_cluster = "dmap-import"
+    environment_name = os.environ["ENVIRONMENT"]
+
+    # get all of the tasks running on the cluster
+    task_arns = client.list_tasks(cluster=dmap_ecs_cluster)["taskArns"]
+
+    # if tasks are running on the cluster, get their descriptions and check to
+    # see if any match the group for this task.
+    #
+    # if the group matches, raise an exception that will terminate the process
+    if task_arns:
+        running_tasks = client.describe_tasks(
+            cluster=dmap_ecs_cluster, tasks=task_arns
+        )["tasks"]
+
+        for task in running_tasks:
+            if f"family:dmap-import-{environment_name}" == task["group"]:
```

Comment:
i'm building this string out based on the environment. in theory, we could also use the task arn directly, but i don't think we can pass that in via terraform because its not yet known.